src/Livewire/StepUpAuthenticationModal.php

```
@@ -0,0 +1,185 @@
+            }
+        } else {
+            $this->error = 'Two-factor authentication is not configured.';
+            $this->code  = '';
+            return;
+        }
+
+        $this->completeStepUp();
+    }
+
+    #[On( 'webauthn-step-up-complete' )]
+    public function verifyWebAuthn( array $response ): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$response'. (undefined)

(UnusedFormalParameter)